content/use-case-apps.md

```diff
@@ -0,0 +1,422 @@
+---
+title: "There's an Apps for that"
```

Comment:
```suggestion
title: "There's an App for that!"
```
I see why you picked this title, just a note it might be bad for SEO

Reply:
Yes, Apple's reference - don't have strong feeling about it, especially if not SEO friendly

Reply:
(applied typo)